solaris/solaris10/README.md

```diff
@@ -8,20 +8,20 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
 To build a Wazuh package you need to install the following tools:
   - `git`: `pkgutil -i git`.
 
-## Building RPM packages
+## Building Solaris11 packages
```

Comment:
Solaris 10 or 11?

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893